### PR TITLE
LPS-192638 | master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-192638](https://liferay.atlassian.net/browse/LPS-192638)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:close

**Comment by victorhcunha** [comment]
ci:open

**Comment by victorhcunha** [comment]
ci:reopen

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by victorhcunha** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+		ObjectAdmin.deleteAllCustomObjectsViaAPI();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-182586 - Verify it's possible to define output fields for my validation as partial validation"
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:24]: Verify an error message is seen for an invalid field in an object entry if the validation is set to partial validation

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+	@priority = 5
+	test CanAddEntry {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:26]: CanValidateField

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+		property custom.properties = "feature.flag.LPS-187846=true";
+		property portal.acceptance = "true";
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:28]: Let's not add this to acceptance until we're sure this is stable.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "BigDecimal",
+				isRequired = "false",
+				objectName = "CustomObject192638");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject192638");
+		}
+
+		task ("and Given an expression builder to validate if does not contain some numbers and set to partial validation with the precision decimal") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:47]: and Given an expression builder validation set to partial validation with the precision decimal field

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+			Click(locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE");
+
+			Click(
+				key_columnOption = "Custom Field",
+				locator1 = "ObjectCustomViews#SELECT_AUTO_COMPLETE_OPTION");
+
+			PortletEntry.save();
+		}
+
+		task ("When adding an object entry with incorrect validation") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:81]: When the user attempts to create an invalid object entry

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject192638");
+
+			ObjectPortlet.addSingleFieldEntryViaUI(entry = 123);
+		}
+
+		task ("Then assert the error message in the object entries layout to the decimal field") {
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:87]: Then an error message is seen and the object entry isn't added

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+			Alert.viewErrorMessage(errorMessage = "Error:This entry is not possible.");
+		}
```
Comment (timpak) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase:89]: Please include an assertion to verify the entry isn't added

**Review by timpak — CHANGES_REQUESTED** [review]
@victorhcunha please see my comments below

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**